src/tools/cephfs/TableTool.cc

```diff
@@ -261,14 +261,112 @@ class TableHandler
   }
 };
 
+template <typename A>
+class TableHandlerOmap
```

Comment:
New test for sessionmap persistence is piggybacked on my wip-10390 ceph-qa-suite branch